apps/examples/aws-example/config/app-config.json

```diff
@@ -0,0 +1,71 @@
+{
+  "app": {
+    "name": "aws-example",
+    "version": "${HOPEIT_APPS_API_VERSION}"
+  },
```

Comment:
@leosmerling-hopeit what strategy do you think for the versioning of the plugin?

Reply:
0.1